src/core/Akka.Remote/MessageSerializer.cs

```diff
@@ -41,16 +42,16 @@ public static object Deserialize(ActorSystem system, SerializedMessage messagePr
         /// <returns>SerializedMessage.</returns>
         public static SerializedMessage Serialize(ActorSystem system, Address address, object message)
         {
-            Serializer serializer = system.Serialization.FindSerializerFor(message);
```

Comment:
IMHO, throwing an exception if we try to serialize a null message is fine by me. We can't send it anyway :p